pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -369,6 +371,15 @@ func (v *vsphereVolumeProvisioner) Provision(selectedNode *v1.Node, allowedTopol
 		volSpec.Fstype = "ext4"
 	}
 
+	var volumeMode *v1.PersistentVolumeMode
```

Comment:
Do we need to remove following check from ` func (v *vsphereVolumeProvisioner) Provision`? 

```
if util.CheckPersistentVolumeClaimModeBlock(v.options.PVC) {
 		return nil, fmt.Errorf("%s does not support block volume provisioning", v.plugin.GetPluginName())
 	}
```

Reply:
Yes, we need to remove it! Good catch!